src/pages/founding-members/components/List/index.js

```diff
@@ -1,55 +1,56 @@
-import React from 'react';
+import React, { useState } from 'react';
 import CardCarousel from '../../../../components/CardCarousel';
 
 import './style.scss';
 
-const Card = ({ founderData: { main, referrer, score, referralScore } }) => (
-  <div className="FoundingMembersPage__card">
-    <div className="FoundingMembersPage__card__main">
-      {main.icon ? (
-        <img className="FoundingMembersPage__card__main__icon" src={main.icon} alt='icon of founding member'/>
-      ) : (
-        <div className="FoundingMembersPage__card__main__icon"></div>
-      )}
-      <div className="FoundingMembersPage__card__main__content">
-        <p className="FoundingMembersPage__card__main__name">{main.name}</p>
-        <p className="FoundingMembersPage__card__main__handle">{main.handle}</p>
-      </div>
-    </div>
-    <div className="FoundingMembersPage__card__scores">
-      <div className="FoundingMembersPage__card__score">
-        <p className="FoundingMembersPage__card__score__title">Direct score</p>
-        <p className="FoundingMembersPage__card__score__stat">{score}</p>
-      </div>
-      <div className="FoundingMembersPage__card__score">
-        <p className="FoundingMembersPage__card__score__title">Referral score</p>
-        <p className="FoundingMembersPage__card__score__stat">{referralScore}</p>
+const Card = ({ founderData, partialTokenAllocation }) => {
+  const [imageHasError, setImageHasError] = useState(false);
+
+  const tokensAllocated = founderData?.extraAllocation + (founderData?.totalScore * partialTokenAllocation);
```

Comment:
I was unclear in previous comments, but there should be two values here:
```
const tokensAllocated = founderData?.extraAllocation
const tokensProjected = (founderData?.totalScore * partialTokenAllocation)
```

The former is what they have already earned, whereas the latter is only a projection. Example:

For `member1`
tokensProjected = 0.02
tokensAllocated = 3

I was initially thinking it should just be:
Tokens allocated
0.02% + 3% *

but for clarity:

Tokens allocated / projected
0.02% / 3%

Reply:
This is just in the List (carousels). I assume we want it in all the other places as well. Correct?

Reply:
Yes, I was just was using this as an example...